### PR TITLE
feat(Kafka/Parquet): added processing time param to streaming output …

### PR DESCRIPTION
**Pull request by RonBarabash**
…mode

### CODE REVIEW

File: src/main/scala/com/yotpo/metorikku/output/writers/kafka/KafkaOutputWriter.scala [hunk 1]
```diff
@@ -56,12 +61,14 @@ class KafkaOutputWriter(props: Map[String, String], config: Option[Kafka]) exten
         log.info(s"Writing Dataframe to Kafka Topic ${kafkaOptions.topic}")
         val df: DataFrame = selectedColumnsDataframe(dataFrame)
         val stream = df.writeStream.format("kafka")
+          .trigger(Trigger.ProcessingTime(kafkaOptions.processingTime))
```
Comment (lyogev) [inline src/main/scala/com/yotpo/metorikku/output/writers/kafka/KafkaOutputWriter.scala]: Why not allow full control of the trigger? Why force a specific trigger.
Also let's add support for watermarking if we're doing this fix

File: examples/kafka/kafka2kafka.yaml [hunk 2]
```diff
@@ -12,3 +12,6 @@ output:
     topic: testAppendOutputMode
     valueColumn: value
     outputMode: append
+  trigger:
+      triggerType: ProcessingTime
```
Comment (lyogev) [inline examples/kafka/kafka2kafka.yaml]: move inside outputoptions and remove trigger group

**Comment by RonBarabash** [comment]
@lyogev  please review

File: src/main/scala/com/yotpo/metorikku/metric/config/Output.scala [hunk 3]
```diff
@@ -1,4 +1,4 @@
 package com.yotpo.metorikku.metric.config
 
-case class Output(dataFrameName: String, outputType: String, outputOptions: Map[String, Any]) {
+case class Output(dataFrameName: String, outputType: String, outputOptions: Map[String, Any], trigger: Map[String, Any]) {
```
Comment (lyogev) [inline src/main/scala/com/yotpo/metorikku/metric/config/Output.scala]: remove trigger

**Pull request merged**